### DISCUSSION TITLE
Use measured plasma density profile as input

### DISCUSSION THREAD

**Discussion by berceanu** [BLAST-WarpX/warpx#4382] [Simulation Inputs]
Does `WarpX` support using an experimentally measured plasma density profile as simulation input for LWFA scenarios?

**Comment by ax3l** [comment]
Not yet:
- https://github.com/ECP-WarpX/WarpX/issues/3948
- https://github.com/ECP-WarpX/WarpX/issues/618

**Comment by pordyna** [comment]
@ax3l Hi! I would be also interested in this feature, but I have seen that one can initialize particles from openPMD. So I guess this would be possible to precalculate positions of all particles in beforehand and pass it via openpmd? Not a very efficient way, but would it work as a workaround?
